specification/dimensions/resourcetype.md

```diff
@@ -0,0 +1,40 @@
+# Resource Type
+
+Resource Type describes the kind of resource for which you are being charged.  A Resource Type is commonly used for scenarios like identifying cost changes in groups of similar resources.
+
+Resource Type MUST be present within billing data.  Resource Type MUST be of type String and MUST NOT be NULL when a corresponding ResourceId is not NULL.  When a corresponding ResourceId value is NULL, the ResourceType column value MUST also be NULL.  Resource Type MUST contain consistent values and a consistent value-format that accounts for additional values added.
+
+Some potential values MAY include Virtual Machine, Data Warehouse, and Load Balancer.
```

Comment:
I really don't see the value in this. Either we provide a recommended set or we leave this out of normative content. If we really want to show some example values, how about between sentence one and two in the first paragraph?

If we want to put in normative content, we should probably provide a more comprehensive list and have something stronger like if the resource is one of these types, these values MUST be used. It doesn't seem that's the intention here.